src/components/Charts/AreaChart/AreaChart.js

```diff
@@ -222,16 +222,22 @@ export default class AreaChart extends React.Component {
 }
 
 AreaChart.propTypes = {
```

Comment:
following changes will not pass `eslint` check, because object keys are not in alphabetical order. If it is important for us to have specific props order, then I would suggest to turn of this requirement for specific part of code:

```
/* eslint-disable sort-keys */
AreaChart.propTypes = {
  height: PropTypes.number,
  // ...
};
/* eslint-enable sort-keys */
```